repos/kaleido/cc/scopes/Mermaid.h

```diff
@@ -0,0 +1,87 @@
+//
+// Created by coma007 on 4/4/24.
+//
+#include "Base.h"
+#include "base/bind.h"
+#include "base/command_line.h"
+#include "base/strings/string_util.h"
+#include "base/strings/stringprintf.h"
+#include "headless/public/devtools/domains/runtime.h"
+#include "../utils.h"
+#include <streambuf>
+#include <string>
+#include <sstream>
+#include <iostream>
+#include <fstream>
+
+#ifndef CHROMIUM_MERMAIDSCOPE_H
+#define CHROMIUM_MERMAIDSCOPE_H
+
+namespace kaleido {
+    namespace scopes {
+
+        class MermaidScope : public BaseScope {
+        public:
+            MermaidScope();
+
+            ~MermaidScope() override;
+
+            MermaidScope(const MermaidScope &v);
+
+            std::string ScopeName() override;
+
+            std::vector<std::unique_ptr<::headless::runtime::CallArgument>> BuildCallArguments() override;
+
+        };
+
+        MermaidScope::MermaidScope()  {
+
+            // Initialize mermaid object code
+            std::string mermaidInit = "mermaid.initialize( { startOnLoad: false, securityLevel: 'loose'} ); window.mermaid = mermaid;";
```

Comment:
Can we also pass this config object used for initialization as an argument to the render function, and then call mermaid.initialize() inside the render function?

Reply:
I think that I've already tried that and thought that it did not work because it was not placed in `<head>` tag. I will try again, because previous time I had that problem with mermaid versions.

Reply:
`mermaid.intialize` must be called right after the import, in the same `script` tag. 

I tried to call it in the `<head>`, but in a `<script>` after the import, it did not work.
I tried to pass it to the `render` (I suppose it is in `<body>` then), it did not work.